core-services/src/test/java/org/zalando/nakadi/service/publishing/AvroEventPublisherTest.java

```diff
@@ -0,0 +1,41 @@
+package org.zalando.nakadi.service.publishing;
+
+
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.service.AvroSchema;
+import org.zalando.nakadi.util.UUIDGenerator;
+
+import static org.zalando.nakadi.utils.TestUtils.buildDefaultEventType;
+
+public class AvroEventPublisherTest extends EventPublisherTest {
+
+    @Test
+    public void testAvroEventWasSerialized() throws Exception {
+        final Resource metadata = new DefaultResourceLoader().getResource("metadata.avsc");
+        final Resource accessLog = new DefaultResourceLoader().getResource("nakadi.access.log.avsc");
+        final AvroSchema avroSchema = new AvroSchema(metadata, accessLog);
+        final AvroEventPublisher eventPublisher = new AvroEventPublisher(timelineService,
+                cache, timelineSync, nakadiSettings, new UUIDGenerator(), avroSchema);
+        final EventType eventType = buildDefaultEventType();
+        Mockito.when(cache.getEventType(eventType.getName())).thenReturn(eventType);
+
+        final GenericRecord event = new GenericData.Record(avroSchema.getNakadiAccessLogSchema());
+        event.put("method", "POST");
+        event.put("path", "/evnet-types");
```

Comment:
```suggestion
        event.put("path", "/event-types");
```
Doesn't matter in tests, but doesn't look good either.